test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -615,12 +615,13 @@ function start-kubemaster-component() {
 			local audit_policy_config_mount=""
 			local audit_policy_config_volume=""
 			if [[ "${ENABLE_APISERVER_ADVANCED_AUDIT:-}" == "true" ]]; then
-				read -d '' audit_policy_config_mount << EOF
+				# FIXME: This is failing
```

Comment:
I don't know yet if this version will be working, but without any code changes it doesn't work.

@shyamjvs - I think you did that, do you know what's the best way to fix this.

Reply:
Could you clarify how you know it's failing here?
From the kubemark-master kubelet logs it seems like kubelet didn't even try to create apiserver pod - https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/73578/pull-kubernetes-kubemark-e2e-gce-big/36412/artifacts/e2e-73578-ac87c-kubemark-master/kubelet.log

Reply:
Yes - because this instruction is failing and -o exit is causing the script to stop at this point (so we don't have manifest pod for apiserver in the end).

Reply:
Right.. sorry I misunderstood.

> @shyamjvs - I think you did that, do you know what's the best way to fix this.

Unfortunately no. Can you change this PR to print the error when running this script? Currently I'm unable to run a kubemark cluster from my end.

Reply:
@wojtek-t Sorry for big delay.. had a hectic last few days. It seems like replacing this part of code with https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L1618-L1668 should work. Can you try or I can help with that too?

Reply:
If you could help with that it would be great - I'm not sure I will get to this anytime soon.